src/main/scala/org/broadinstitute/hail/methods/Aggregators.scala

```diff
@@ -41,10 +42,15 @@ object Aggregators {
     } else None
   }
 
-  def buildSampleAggregations(vds: VariantDataset, ec: EvalContext): Option[(String) => Unit] = {
+  def buildSampleAggregations(vds: VariantDataset, ec: EvalContext, branchingFactor: Option[Int]): Option[(String) => Unit] = {
```

Comment:
What default argument?  I thought about this but punted because I couldn't decide.